internal/resources/grafana/resource_organization_test.go

```diff
@@ -110,6 +113,9 @@ func TestAccOrganization_users(t *testing.T) {
 					resource.TestCheckNoResourceAttr(
 						"grafana_organization.test", "viewers.#",
 					),
+					resource.TestCheckNoResourceAttr(
+						"grafana_organization.test", "nones.#",
```

Comment:
I think this will need a test where there is one in the list

Reply:
Sounds good!
Will do, thanks!